### PR TITLE
May k3s release

### PR DESCRIPTION
**Pull request by nikolaishields**

### CODE REVIEW

**Review by briandowns — DISMISSED** [review]

File: channels.yaml [hunk 1]
```diff
@@ -203,3 +213,8 @@ releases:
     maxChannelServerVersion: v2.6.99
     serverArgs: *serverArgs-v2
     agentArgs: *agentArgs-v2
+  - version: v1.23.7-rc1+k3s1
+    minChannelServerVersion: v2.6.4-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: *serverArgs-v2
```
Comment (rancher-max) [inline channels.yaml]:
```
serverArgs: &serverArgs-v3
      <<: *serverArgs-v2
      egress-selector-mode:
        type: string
```

**Review by rancher-max — APPROVED** [review]

File: channels.yaml [hunk 2]
```diff
@@ -201,5 +211,12 @@ releases:
   - version: v1.23.6+k3s1
     minChannelServerVersion: v2.6.4-alpha1
     maxChannelServerVersion: v2.6.99
-    serverArgs: *serverArgs-v2
+    agentArgs: *agentArgs-v2
```
Comment (rancher-max) [inline channels.yaml:218]: Ahhh wait, it looks like serverargs got removed here by accident?

**Review by rancher-max — CHANGES_REQUESTED** [review]

File: channels.yaml (hunk 2 above)
Comment (nikolaishields) [reply channels.yaml:218]: Good catch!

**Review by rancher-max — DISMISSED** [review]

**Review by rancher-max — DISMISSED** [review]

**Review by rancher-max — APPROVED** [review]
LGTM

**Review by briandowns — APPROVED** [review]

**Pull request merged**